app/models/concerns/product_sort_by_stocks.rb

```diff
@@ -31,5 +41,9 @@ class << self
     ransacker :backorderable_priority do
       @backorderable_priority_sql
     end
+
+    ransacker :backorderable_name do
+      @backorderable_name_sql
+    end
```

Comment:
Interesting, I would have tried to give the expression a column alias name (that is, append `as backorderable_name` to the expression) and refer to it by name, instead of repeating the SQL here.

But I can see this is how the other ransackers are already set up, so maybe there's a reason for that? In any case, this seems fine 👍